/**
 * Copyright (c) 2025 Huawei Technologies Co., Ltd.
 * This program is free software, you can redistribute it and/or modify it under the terms and conditions of
 * CANN Open Software License Agreement Version 2.0 (the "License").
 * Please refer to the License for details. You may not use this file except in compliance with the License.
 * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED,
 * INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
 * See LICENSE in the root of the software repository for the full text of the License.
 */
 
syntax = "proto3";
package platform.tiling;
 
message PlatformInfosDef {
map<string, ListDType> ai_core_intrinsic_dtype_map = 1;
map<string, ListDType> vector_core_intrinsic_dtype_map = 2;
map<string, PlatformMap> platform_res_map = 3;
map<string, ListDType> fixpipe_dtype_map = 4;
}
message ListDType {
repeated string dtype = 1;
}
message PlatformMap {
map<string, string> platform_map = 1;
}